pkg/registry/service/registry.go

```diff
@@ -30,21 +33,22 @@ type Registry interface {
 	DeleteService(ctx api.Context, name string) error
 	UpdateService(ctx api.Context, svc *api.Service) (*api.Service, error)
 	WatchServices(ctx api.Context, options *api.ListOptions) (watch.Interface, error)
+	ExportService(ctx api.Context, name string, options unversioned.ExportOptions) (*api.Service, error)
 }
 
 // storage puts strong typing around storage calls
 type storage struct {
-	rest.StandardStorage
+	ss rest.StandardStorage
```

Comment:
Please leave unnamed, or if you have to name it, use a name like 'delegate' or something else more descriptive. Tiny names are good for local variable names in short functions, but not good as members in structs.